.gitignore

```diff
@@ -28,3 +28,6 @@ adapters/solana/libyellowstone_grpc_geyser.dylib
 reveal_*.tx
 
 *.txs
+
+elf/
+dbs
```

Comment:
I think putting dbs in gitignore broke something, because we recently added and removed it